packages/CollabCodeEdit-Core.package/CCEMessage.class/class/recieveEdit..st

```diff
@@ -0,0 +1,4 @@
+server-Message creation
+recieveEdit: aCCEEdit
+	^ self new: (CCEMessage serverMsgTypes at: #recieveEdit:from:) 
+			with: aCCEEdit
```

Comment:
There are no senders of receiveEdit. Therefore i would suggest to delete the method. its functionality is already covered in forewardEditMessage

Reply:
which should be renamed anyway:D